aqua/wezterm/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cargo 1.0
 
-github.setup        wez wezterm 20210405-110924-a5bb5be8
+github.setup        wez wezterm 20210502-154244-3f7122cb
 revision            1
```

Comment:
When the version gets bumped, `revision` resets back to 0.

Let's set that and we should be good to go here.

Reply:
Done! Thanks for the heads up; first time poking at MacPorts :)